src/screens/SearchScreen.tsx

```diff
@@ -48,6 +47,7 @@ export default function SearchScreen({
           placeholder="Search"
           onChangeText={onSearchChange}
           value={searchQuery}
+          autoComplete={undefined}
```

Comment:
seems to be an issue with react-native-paper itself, typing enforces autoComplete as a mandatory key -- https://github.com/callstack/react-native-paper/issues/1971